backend/img_upload.ts

```diff
@@ -0,0 +1,45 @@
+import dotenv from 'dotenv'
+import axios from 'axios'
+import FormData from 'form-data'
+import fs from 'fs'
+
+// initialize configuration
+dotenv.config()
+
+const PINATA_JWT = process.env.PINATA_JWT ?? ''
+const PINATA_PINFILE_URL = process.env.PINATA_PINFILE_URL ?? ''
```

Comment:
Add these const to your `.env` file

- `PINATA_JWT=...`
- `PINATA_PINFILE_URL=...`